Xamarin.Forms.Platform.iOS/Renderers/ListViewRenderer.cs

```diff
@@ -685,7 +695,7 @@ void UpdateSeparatorColor()
 			// ...and Steve said to the unbelievers the separator shall be gray, and gray it was. The unbelievers looked on, and saw that it was good, and
 			// they went forth and documented the default color. The holy scripture still reflects this default.
```

Comment:
and still it does. behold 
![image](https://user-images.githubusercontent.com/7827070/77458039-c7b39080-6dba-11ea-8f84-fccca74b8d7e.png)